internal/common/types.go

```diff
@@ -242,6 +252,7 @@ const (
 // getArchMapping is a helper function that defines the conversion from JSON string value
 // to Distribution.
 func getDistributionMapping() map[string]int {
+	// Don't forget to add module-platform-id mapping below
 	mapping := map[string]int{
 		"fedora-30": int(Fedora30),
 		"fedora-31": int(Fedora31),
```

Comment:
F32 should go there as well

Reply:
It's there as part of other PRs.

Reply:
=> it's already there because I rebased to the latest master :)